### PR TITLE
Add _patch.py to showcase additions to generated code

### PR DESCRIPTION
**Pull request by t0mk**
This PR shows what's added to generated code to make it more user-friendly. The code is then used as described in README:

```python
import equinixmetalpy
import os

client = equinixmetalpy.Client(os.getenv("PACKET_AUTH_TOKEN"))

# select first free project
projects_resp = client.find_projects()
pid = projects_resp.projects[0].id

dci = equinixmetalpy.models.DeviceCreateInMetroInput(
    operating_system="ubuntu_18_04",
    plan="c3.small.x86",
    hostname="test",
    metro="sv",
    billing_cycle="hourly",
    tags=["test"]
)

new_device_resp = client.create_device(pid, dci)

# The variable can contain either new device object or error.
# Check it with raise_if_error.
equinixmetalpy.raise_if_error(new_device_resp)

print("New Device:")
print(new_device_resp)
```

Please give feedback on the form.

### CODE REVIEW

File: src/equinixmetalpy/_patch.py [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+            _LOGGER.debug("Failed to log response: %s", repr(err))
+        _LOGGER.debug(get_bottom_divider(topdiv))
+
+
+class Client(GeneratedClient):
+    def __init__(self, credential: str, base_url: str = EM_API, **kwargs: Any) -> None:
+        azure_credential = AzureKeyCredential(credential)
+        if os.getenv("METAL_PYTHON_DEBUG") == "1":
+            kwargs["logging_policy"] = HttpLoggingPolicy(api_endpoint=base_url)
+        sdk_moniker = f"pydo/{_version.VERSION}"
```
Comment (displague) [inline src/equinixmetalpy/_patch.py]: Anything `pydo` should be Equinix Metal'ified.

File: src/equinixmetalpy/_patch.py [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+        super().__init__(azure_credential, base_url, sdk_moniker=sdk_moniker, **kwargs)
```
Comment (displague) [inline src/equinixmetalpy/_patch.py]: anything `azure` / `Azure` should be Equinix'ified

**Comment by displague** [comment]
Can we include the PR description script in an examples directory (wherever that would make sense for a Python project -- `examples/devices`, perhaps as pydoc?)

Should the example be less magical? Require a project id.

Should we set the termination / end time on the device and also fetch the device and delete the device as part of the example?

`PACKET_AUTH_TOKEN` -> `METAL_AUTH_TOKEN`

File: src/equinixmetalpy/_patch.py (hunk 1 above)
Comment (t0mk) [reply src/equinixmetalpy/_patch.py]: thanks. I grepped the repo for pydo, it should be fine by now.

**Comment by t0mk** [comment]
@displague I added a more complete example to `examples/` dir in https://github.com/t0mk/equinixmetalpy/pull/2/commits/21bcbf74c4724a54f9f42312cde349c7dbdbabfa

**Pull request merged**